scripts/zones/Konschtat_Highlands/mobs/Amber_Quadav.lua

```diff
@@ -7,4 +7,7 @@ require("scripts/globals/regimes")
 
 function onMobDeath(mob, player, isKiller)
     tpz.regime.checkRegime(player, mob, 83, 1, tpz.regime.type.FIELDS)
+    if player:getCharVar("TutorialProgress") == 7 then
```

Comment:
Should we move this to a function inside of the global onmobdeathex?

Reply:
if you did a zone check, that check would still exec to determine true/fall for _every kill game wide._

drk quest kills can be done on any mob in the entire game so that does make sense. it could have been done like breaking a latent except for the 2nd quest that includes kills from the first one making that annoying. In the future maybe a pseudo additional effect on that item. But I'm rewriting those and that's a global too after that, so I don't think you want to go from _every mob killed_ into _every swing_ checking quest stuff.

The best answer for tutorial mobs is probably to either wrap them like regimes do, or use a mixin :disappointed: